tools/taskcluster/build_targets.py

```diff
@@ -0,0 +1,88 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+"""
+This script configures what APKs are built using the options passed in
+from the taskcluster {{ event.version }}.
+
+{{ event.version }} should be in the form: <tag name>:[all,release,debug]-<platform name>:[arm,arm64,x86]
+Some examples of {{ event.version }} and the resultant output from this script
```

Comment:
nit: do you want to end this line a period or colon?

Reply:
Period, It was a colon but I thought that was weird since each example has a colon too.